draft-mouris-cfrg-mastic.md

```diff
@@ -237,9 +285,98 @@ This document uses the following terms as defined in {{!VDAF}}:
 "prep share", and
 "report".
 
-Mastic uses finite fields as specified in {{Section 6.1 of !VDAF}}. We usually
-denote a finite field by `F` and its Python class object, a subclass of
-`Field`, as `field: type[F]`.
+Mastic uses finite fields of prime order as specified in {{Section 6.1 of
+  !VDAF}}. We usually denote a finite field by `F` and its Python class object,
+  a subclass of `Field`, as `field: type[F]`. The following basic API functions
+  referenced in this document are as defined therein:
+
+* `Field(integer: int) -> Field` returns an integer represented as a field
+  element. The value of integer MUST be non-negative and less than
+  Field.MODULUS.
+* `Field.as_unsigned() -> int` returns the integer representation of a field
+  element.
+* `Field.encode_vec(vec: list[Field]) -> bytes` encodes a vector of field
+  elements `vec` as a byte string.
+* `Field.zeros(length: int) -> list[Self]` returns a vector of zeros of size
+  `length`.
+
+The following parameters are also referenced:
+
+  | Parameter           | Description                                           | Value                   |
+  |:--------------------|:------------------------------------------------------|:------------------------|
+  | `field: type[F]`    | class object for the field ({{Section 6.1 of !VDAF}}) | set by constructor      |
+  | `Field.MODULUS: int`| prime modulus defining the field arithmetic           | to be defined           |
+  {: #field-params title="Basic Field parameters."}
```

Comment:
What's the purpose of this table? I don't see how its contents relate to one another. For example, `field` is an attribute of a validity circuit, for instance.